### PR TITLE
Fixed runtime exception

### PR DESCRIPTION
**Pull request by hamsternik**
Fixed runtime exception on the login screen.

### CODE REVIEW

File: share.android.js [hunk 1]
```diff
@@ -14,7 +14,8 @@ export default class Share extends Component {
 
     this.state = {
       isOpen: true,
-      server: null, token: null, type: null, url: null,
+        server: null, token: null, type: null,
```
Comment (drets) [inline share.android.js]: Could you align code properly?

Comment (hamsternik) [reply share.android.js]: Done.

File: share.android.js [hunk 2]
```diff
@@ -14,8 +14,8 @@ export default class Share extends Component {
 
     this.state = {
       isOpen: true,
-        server: null, token: null, type: null,
-        url: null,
+      server: null, token: null, type: null,
+      url: null
```
Comment (drets) [inline share.android.js]: Not sure, but absence of trailing comma may fail a linter.

Comment (hamsternik) [reply share.android.js]: Maybe. I have fixed this one)

File: share.android.js [hunk 3]
```diff
@@ -14,7 +14,8 @@ export default class Share extends Component {
 
     this.state = {
       isOpen: true,
-      server: null, token: null, type: null, url: null,
+      server: null, token: null, type: null,
+      url: null,
```
Comment (rasendubi) [inline share.android.js]: Please don't break the line. This will cause merge conflict with my other PR.

**Comment by drets** [comment]
After this change − please squash to the one commit.

File: share.android.js (hunk 3 above)
Comment (drets) [reply share.android.js]: After this change − please squash to the one commit.

**Pull request merged**